src/core/gps/qgsnmeaconnection.cpp

```diff
@@ -108,7 +109,8 @@ void QgsNmeaConnection::processStringBuffer()
           mStatus = GPSDataReceived;
           QgsDebugMsgLevel( QStringLiteral( "*******************GPS data received****************" ), 2 );
         }
-        else if ( substring.startsWith( QLatin1String( "$GPGSV" ) ) || substring.startsWith( QLatin1String( "$GNGSV" ) ) )
+        // GPS+SBAS GLONASS GALILEO BEIDOU;
+        else if ( substring.startsWith( QLatin1String( "$GPGSV" ) ) || substring.startsWith( QLatin1String( "$GNGSV" ) ) || substring.startsWith( QLatin1String( "$GLGSV" ) ) || substring.startsWith( QLatin1String( "$GAGSV" ) ) || substring.startsWith( QLatin1String( "$GBGSV" ) ) )
```

Comment:
See https://github.com/qgis/QGIS/pull/49820/files#r957631171

Reply:
Personally I would leave the possibility to someone else to modify that line 113 introducing $ GQGSV and eventually open a new Pull Request on this specific NMEA message.

Reply:
I changed my mind!
Perhaps in my NMEA logs the $ GQGSV message is "distorted" because the QZSS constellation is almost always low on the horizon in my area and therefore, not having good reception, the relative NMEA message is not complete!
Generally in my GNSS receiver, the satellites of the QZSS constellation are not enabled, I don't use them!
it is better to introduce "$ GQGSV" immediately and if something does not work there will be a new Issues!
The only way to check the position of the satellites would be to enable the SkyPlot satellites, you have to hear @nyalldawson if you can!